src/lib/class/Client/Client.cpp

```diff
@@ -105,3 +105,10 @@ std::string const Client::get_next_msg()
 		return std::string();
 	return msg;
 }
+
+/**
+ * @brief Generate a prefix corresponding to a specific client.
+ *
+ * @return the newly created prefix.
+ */
```

Comment:
Missing the `@throw` section, because the operator `+` for `std::string` may fail.